pandas/core/series.py

```diff
@@ -2289,6 +2289,11 @@ def combine(self, other, func, fill_value=None):
         Series
             The result of combining the Series with the other object.
 
+        See Also
+        --------
+        Series.combine_first : Combine Series values, choosing the calling
+                               Series' values first.
```

Comment:
The indentation is wrong, should be indented 4 spaces respect to the previous line.